### ISSUE TITLE
Price problems

### ISSUE THREAD

**Issue by hoangvu13** [spartan-protocol/SpartanProtocol-DAppV2#537]
1. Price is wrong from market on Sparta/BNB pair at 12.09AM(GMT+7)-27/9/2021
3.87 BNB x 349 USD = 1,350 while Sparta displays ~ 950USD
![image](https://user-images.githubusercontent.com/82772140/135063163-8dcac3c3-67ef-4e3d-ab0d-4fce3112c847.png)
![image](https://user-images.githubusercontent.com/82772140/135063206-58faa5f1-8bd7-4d6e-9c29-e7803668a165.png)
2. The ratio Sparta/BUSD and Sparta/DAI are skewed quite large ~ 35%
![image](https://user-images.githubusercontent.com/82772140/135063244-c9fa3c13-c852-4722-9547-c21d002fca79.png)
![image](https://user-images.githubusercontent.com/82772140/135063247-198cb6ce-e7c6-4e5b-b626-f5bbc967076a.png)
TG: @HoangVu131
Address: 0xbFCbbbB78239818485AF0B4deBc23cAabf56E0EA
Testnet time: 27/9- 28/9/2021

**Comment by SamusElderg** [comment]
Appreciate the feedback thanks buddy.
Keep in mind this is how the AMM works. The two assets in the pool provide a ratio that can be calc-ed against the SPARTA price and used for internal pricing (kinda the whole point of the protocol)

On testnet however the assets don't have any real market value so people aren't economically incentivized to balance the pools to the agreed market rate so the prices will not match the outside market for the mainnet assets that these testnet assets are merely pretending to be pegged to.

So in short; this isn't a bug; it's the AMM doing expected behaviour. I expect at some point for there to be some sort of an icon/info tooltip to make it more clear that these prices are calculated via internal pool pricing with a link to an article explaining AMMs or similar though so maybe that will help with any future confusion 👍
